lib/screenplay_web/controllers/config_controller.ex

```diff
@@ -50,16 +68,24 @@ defmodule ScreenplayWeb.ConfigController do
             false
         end
 
-        live_screens = ScreensConfigCache.screens(filter_fn)
-        pending_screens = PendingScreensConfigCache.screens(filter_fn)
+        live_screens =
+          ScreensConfigCache.screens(filter_fn)
+          |> Enum.map(fn {k, v} -> {k, Screen.to_json(v)} end)
+          |> Enum.into(%{})
+
+        pending_screens =
+          pending_screens_config
+          |> Enum.filter(&filter_fn.(&1))
```

Comment:
```suggestion
          |> Enum.filter(filter_fn)
```

Reply:
Ahh, thank you. I was having an embarrassingly hard time figuring out how to pass an anonymous function variable 🤦